packages/shared/src/parsers.ts

```diff
@@ -91,14 +93,19 @@ export const ethereumAddress = hexString.refine(
 );
 
 export const chainflipAddress = string.refine(
-  (address) => address.startsWith('cF') && ss58.decode(address),
+  (address) => ss58.decode(address).ss58Format === chainflipSS58Prefix,
   (address) => ({ message: `${address} is not a valid Chainflip address` }),
 );
 
 export const solanaAddress = string.refine(isValidSolanaAddress, (address) => ({
   message: `${address} is not a valid Solana address`,
 }));
 
+export const polkadotAddress = string.refine(
+  (address) => ss58.decode(address),
```

Comment:
still need to limit length to 66

Reply:
`decode` checks that the address is 32 bytes. `hexString` doesn't have any limitation, which means that we don't check that EVM addresses either, i can add it for EVM too

Reply:
ah didn't realise decode checked it 👍